src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -189,39 +189,41 @@ end
 end
 
 # TODO: bring up to speed the PCG to remove this error
-validate_global_grid(global_grid) = 
+validate_poisson_solver_distributed_grid(global_grid) = 
         throw("Grids other than the RectilinearGrid are not supported in the Distributed NonhydrostaticModels")
 
-function validate_global_grid(global_grid::RectilinearGrid) 
+function validate_poisson_solver_distributed_grid(global_grid::RectilinearGrid) 
     TX, TY, TZ = topology(global_grid)
 
     if (TY == Bounded && TZ == Periodic) || (TX == Bounded && TY == Periodic) || (TX == Bounded && TZ == Periodic)
-        throw("NonhydrostaticModels on Distributed grids do not support topology ($TX, $TY, $TZ) at the moment.
+        throw("A distributed Poisson solver does not support grids with topology ($TX, $TY, $TZ) at the moment.
```

Comment:
```suggestion
        throw("Distributed Poisson solvers do not support grids with topology ($TX, $TY, $TZ) at the moment.
```

Much better and clearer overall.